check.yml

```diff
@@ -0,0 +1,27 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
```

Comment:
I might misunderstand how we would like to launch it but aren't an E2E runner WF missing here?

Reply:
The code in e2e.go is run directly.